lib/ffprober/ffprobe_version.rb

```diff
@@ -4,7 +4,7 @@ class FfprobeVersion
     NIGHTLY_REGEX = /^(ffprobe|avprobe|ffmpeg) version (N|git)-/
 
     MIN_VERSION = Gem::Version.new('0.9.0')
-    MAX_VERSION = Gem::Version.new('2.3.3')
+    MAX_VERSION = Gem::Version.new('2.5.4')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.